public/lang/fr.ini

```diff
@@ -53,3 +53,19 @@ lang_login_invalid_credentials=Nom d’utilisateur/Mot de passe incorrects
 lang_no_title=Sans titre
 lang_error=Une erreur est survenue
 lang_streamerror=Une erreur s’est produite lors du chargement de la liste, essayer à nouveau ?
+lang_reload_list=Recharger la liste
+lang_sources_refreshed=Les sources ont été rafraichies
+lang_error_session_expired=Votre session a expiré
+lang_error_add_source=Erreur lors de l'ajout de la source
+lang_error_delete_source=Erreur lors de la suppression de la source
+lang_error_load_tags=Erreur lors du chargement des balises
+lang_error_unknown_tag=Étiquette inconnue
+lang_error_unknown_source=Identifiant de source inconnu
+lang_error_mark_items=Impossible de marquer tous les éléments visibles
+lang_error_logout=Impossible de se déconnecter
+lang_error_loading=Erreur de chargement
+lang_error_loading_stats=Impossible de charger les statistiques de navigation
+lang_error_saving_color=Impossible d'enregistrer une nouvelle couleur
+lang_error_refreshing_source=Impossible de rafraichir les sources
+lang_error_sync=Impossible de synchroniser les dernières modifications du serveur
+lang_error_invalid_subsection=Sous-section invalide
```

Comment:
Could you add the final line break? On UNIX systems the lines are terminated by it.

Reply:
Done 😄